scripts/setup

```diff
@@ -41,9 +41,12 @@ cds users create test@test.ch -a --password=123456
 # Create an admin user
 cds users create admin@test.ch -a --password=123456
 cds roles create admin
+cds roles create cern-user
+cds roles add test@test.ch cern-user
 cds roles add admin@test.ch admin
 cds access allow deposit-admin-access role admin
 cds access allow superuser-access role admin
+cds access allow videos-upload-access role cern-user
```

Comment:
remember to apply this when deploying. You might want to already do it in all instances to be sure that you don't forget.